js/ui/diagram/ui.diagram.scroll_view.js

```diff
@@ -2,7 +2,7 @@ import { getWidth, getHeight } from '../../core/utils/size';
 import $ from '../../core/renderer';
 import Widget from '../widget/ui.widget';
 import ScrollView from '../scroll_view';
-import { calculateScrollbarWidth } from '../pivot_grid/utils/calculate_scrollbar_width';
+import { calculateScrollbarWidth } from '../../__internal/grids/pivot_grid/module_widget_utils';
```

Comment:
really weird import, I think we need to move this func in next PRs (hope linter will find this when we'll add rules for imports)